### ISSUE TITLE
NVDA won't install updates on Alpha

### ISSUE THREAD

**Issue by LeonarddeR** [nvaccess/nvda#17564]
### Steps to reproduce:
1. Install NVDA alpha-34867,e2eaab69
2. Check for updates, download the update
3. Press the update now button

### Actual behavior:
An error is in the log and the update isn't installed:

```log
IO - inputCore.InputManager.executeGesture (09:08:10.026) - winInputHook (10064):
Input: kb(laptop):enter
ERROR - core.triggerNVDAExit (09:08:10.031) - MainThread (17692):
NVDA cannot exit safely, ensure open dialogs are closed
ERROR - updateCheck._executeUpdate (09:08:10.042) - MainThread (17692):
NVDA already in process of exiting, this indicates a logic error.
```

### Expected behavior:
No error

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-34867

#### Windows version:
Windows 11 24H2

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Have seen this with more alpha updates recently, starting Wednesday or so.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

**Comment by nidza07** [comment]
Duplicate of #17553 with more info in the comments of that issue, in particular https://github.com/nvaccess/nvda/issues/17553#issuecomment-2555978971

**Comment by LeonarddeR** [comment]
This was actually fixed by #17561
